src/screens/profile/profile.tsx

```diff
@@ -13,7 +17,21 @@ type IProfileScreenProps = Record<string, never>;
 
 const ProfileScreen: React.FC<IProfileScreenProps> = () => {
   const navigation = useNavigation();
-  const [addMenuOpen, setAddMenuOpen] = useState<boolean>(false);
+  const [addMenuOpen, setAddMenuOpen] = useState(false);
+  const [activeIndex, setActiveIndex] = useState(0);
+  const pagerRef = useRef<PagerView>(null);
+  const NAVBAR_ITEMS = [
```

Comment:
Move it outside the component

Reply:
Fix in [357f930](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/9/commits/357f93093cf6dd705105610e61633e33f8b1e92b)